app/assets/stylesheets/application.scss

```diff
@@ -56,6 +55,13 @@ body {
       color: #fff;
       font-size: 16px;
     }
+    ul {
+      list-style-type: none;
+    }
+    li {
+      color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.